docs/superpowers/specs/assets/2026-06-03-cli-device-login-authorize-mockup.html

```
@@ -0,0 +1,176 @@
+          </div>
+          <div class="relative">
+            <select class="appearance-none rounded-lg border border-slate-300 bg-white px-3 py-1.5 pr-8 text-sm text-slate-800 outline-none focus:border-[var(--secondary)]">
+              <option>90 days</option><option>30 days</option><option>7 days</option>
+            </select>
+            <svg class="pointer-events-none absolute right-2.5 top-2 h-4 w-4 text-slate-400" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2"><path d="M6 9l6 6 6-6"/></svg>
+          </div>
+        </div>
+      </div>
+
+      <div class="mt-7 flex flex-col gap-3 sm:flex-row-reverse">
+        <button class="btn-primary-capgo inline-flex w-full items-center justify-center gap-2 rounded-xl px-4 py-3.5 text-base font-semibold">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)